src/System.Management.Automation/engine/hostifaces/MshHostUserInterface.cs

```diff
@@ -1149,6 +1130,15 @@ internal void FlushContentToDisk()
 
                 OutputBeingLogged.Clear();
             }
+
+            Encoding getPathEncoding()
```

Comment:
Please move the local function to start of the method.
Also please use Pascal name case - local function is de-facto a method and could be move in another scope.

Reply:
Done and done. Interesting that you prefer the local function at the start. I haven't seen any recommendation for local functions yet. Do you happen to have coding conventions available for them?

In my team we prefer to put it at the bottom, this way you have "pseudo-code" at the beginning with only the calls and you can just scroll down to see the implementation details.